ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+  Thanks for your interest in using the App Center SDK for React Native.
+
+  If your issue is not about the Analytics/Crashes/Push SDK for React Native,
+  (for example a build issue and you are not using our SDK, or your issue is about CodePush),
+  then please create the ticket on Intercom instead (using the blue chat button on any https://appcenter.ms page).
+
+  Please attach some technical information about your environment,
+  run the following 2 commands and attach output to the ticket:
```

Comment:
attach output to the ticket so that we can investigate the issue
/

Reply:
We can rephrase it like this -

Any issues related to SDK will require you to attach some technical information about your environment. So please run the following two commands and attach the output to your ticket: